ui/job-view/details/summary/ActionBar.jsx

```diff
@@ -187,10 +187,13 @@ class ActionBar extends React.Component {
         staticActionVariables: results.staticActionVariables,
       });
 
-      this.thNotify.send(
-        `Request sent to create an interactive job via actions.json.
-          You will soon receive an email containing a link to interact with the task.`,
-        'success');
+      this.thNotify.send('Request sent to create an interactive job via actions.json.', 'success');
+
+      Object.assign(window.open(), {
```

Comment:
I haven't seen the pattern of passing `window.open()` to `Object.assign()` before. Is there an advantage of using it (as opposed to calling directly)?

Reply:
`window.open(getInteractiveTaskUrl(taskId), '_blank', 'noopener')` opens the link in a new window instead of a tab.

Reply:
It correctly opens a new tab rather than a window in the other places in the codebase where we use `window.open()` for me?
If this pattern is preferred that's ok, but I'd rather be consistent throughout.

Reply:
Opens a new tab: `window.open(url, '_blank')`.
Opens a new window: `window.open(url, '_blank', 'noopener')`;

FYI, probably a separate issue, none of the `window.open` links in the repo have `noopener` in them.

Reply:
Ah right makes sense.

> FYI, probably a separate issue, none of the `window.open` links in the repo have `noopener` in them.

The auth one is intentional/necessary ([code comment](https://github.com/mozilla/treeherder/blob/d9a357ce4e8c83e0daa6113cd576a30d4df62f2e/ui/shared/Login.jsx#L87)) and the same origin ones are fine - however yeah it looks like there are some ones to Bugzilla that we should fix :-)